lua/nvim-treesitter/configs.lua

```diff
@@ -303,7 +303,11 @@ end
 function M.setup(user_data)
   config.modules = vim.tbl_deep_extend('force', config.modules, user_data)
 
-  require'nvim-treesitter.install'.ensure_installed(user_data.ensure_installed)
+  local ensure_installed = user_data.ensure_installed
+  if #ensure_installed > 0 then
```

Comment:
this can be `nil`: `if ensure_installed then` or `if ensure_installed and #ensure_installed > 0` then

Reply:
`local ensure_install = user_data.ensure_isntalled or {}`

Reply:
Changed!